test/org/sagebionetworks/bridge/models/schedules/ScheduleTest.java

```diff
@@ -106,4 +111,55 @@ public void testStringSetters() {
         assertEquals(period, schedule.getInterval());
         assertEquals(Lists.newArrayList(LocalTime.parse("10:10"), LocalTime.parse("12:10")), schedule.getTimes());
     }
+    
+    @Test
+    public void scheduleIdentifiesWhenItIsPersistent() {
+        Schedule schedule = new Schedule();
+        assertFalse(schedule.getPersistent()); // safe to do this before anything else.
+        
+        Survey survey = new TestSurvey(false);
+        Activity activity = new Activity.Builder().withLabel("Test").withSurvey(survey.getIdentifier(),
+                        survey.getGuid(), new DateTime(survey.getCreatedOn())).build();
+        schedule.addActivity(activity);
+        schedule.setScheduleType(ScheduleType.ONCE);
+        schedule.setEventId("survey:"+survey.getGuid()+":finished,enrollment");
+        assertTrue(schedule.getPersistent());
+        
+        schedule.setScheduleType(ScheduleType.RECURRING);
+        assertFalse(schedule.getPersistent());
+        
+        schedule.setScheduleType(ScheduleType.ONCE);
+        schedule.setEventId(null);
+        assertFalse(schedule.getPersistent());
+        
+        schedule = new Schedule();
+        activity = new Activity.Builder().withLabel("Test").withTask("BBB").build();
+        schedule.addActivity(activity);
+        schedule.setScheduleType(ScheduleType.ONCE);
+        schedule.setEventId("enrollment,task:BBB:finished");
+        assertTrue(schedule.getPersistent());
+    }
+    
+    @Test
+    public void scheduleWithDelayNotPersistent() {
+        Schedule schedule = new Schedule();
+        Survey survey = new TestSurvey(false);
+        Activity activity = new Activity.Builder().withLabel("Test").withSurvey(survey.getIdentifier(),
+                        survey.getGuid(), new DateTime(survey.getCreatedOn())).build();
+        schedule.addActivity(activity);
+        schedule.setScheduleType(ScheduleType.ONCE);
+        schedule.setEventId("survey:"+survey.getGuid()+":finished,enrollment");
+        
+        schedule.setDelay("P1D");
+        assertEquals(false, schedule.getPersistent());
```

Comment:
Trivial.  For this and below, instead of assertEquals(false/true, getPersisent()), can we use assertFalse() and assertTrue() directly.